### PR TITLE
Add a test for `tlj` ##test

### PR DESCRIPTION
**Pull request by officialcjunior**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the documentation and the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**
Here, this achieves two things:
* Add a test for `tlj`
* Fix a typo under the help message of `ts?`

**Test plan**

Merge when tests pass.

**Closing issues**

None

### CODE REVIEW

File: doc/fortunes.tips [hunk 1]
```diff
@@ -28,7 +28,7 @@ Add comments using the ';' key in visual mode or the 'CC' command from the rizin
 Assemble opcodes with the 'a' and 'A' keys in visual mode, which are bindings to the 'wa' and 'wA' commands
 Find expanded AES keys in memory with '/ca'
 Find wide-char strings with the '/w <string>' command
-Enable ascii-art jump lines in disassembly by setting 'e asm.lines=true'. asm.lines.out and asm.linestyle may interest you as well
+Enable ascii-art jump lines in disassembly by setting 'e asm.lines=true'. asm.lines.out may interest you as well
```
Comment (ITAYC0HEN) [inline doc/fortunes.tips]: It is there by default. I'd remove this fortune completely

Comment (ITAYC0HEN) [reply doc/fortunes.tips]: Also, might be nor related to this PR ? :D

Comment (ITAYC0HEN) [reply doc/fortunes.tips]: Well, if you just used it for two changes then that's fine. But better to separate it haha

**Review by ITAYC0HEN — CHANGES_REQUESTED** [review]

File: doc/fortunes.tips (hunk 1 above)
Comment (officialcjunior) [reply doc/fortunes.tips]: >It is there by default. I'd remove this fortune completely

Oh, really? Sorry about that. Didn't realize that.
Yeah, just saw that it's turned on by default. 
Will remove that line 👍🏻 

> Also, might be nor related to this PR ? :D

I was about to ask you about that. Should that be another PR or not?

Comment (ITAYC0HEN) [reply doc/fortunes.tips]: usually, it should, but we can skip this time I think :slightly_smiling_face: It's not a big deal

Comment (officialcjunior) [reply doc/fortunes.tips:31]: As discussed, I'll send another PR to fix this.

Comment (wargio) [reply doc/fortunes.tips]: i agree. this should go.

**Comment by wargio** [comment]
@officialcjunior sorry for asking this, but can you make a new PR for the fortunes where you remove all the spaces next to the equals from the tips where it uses the cmd `e keyword = value` ?

like
```
Disable these messages with 'e cfg.fortunes = false' in your ~/.rizinrc
```
becomes
```
Disable these messages with 'e cfg.fortunes=false' in your ~/.rizinrc
```

**Review by XVilka — APPROVED** [review]

**Comment by officialcjunior** [comment]
@wargio Sure, I'll send it right away.

File: doc/fortunes.tips (hunk 1 above)
Comment (officialcjunior) [reply doc/fortunes.tips]: Removing this line on this PR : https://github.com/rizinorg/rizin/pull/359.
I think this conversation can be resolved.

**Review by ITAYC0HEN — APPROVED** [review]
Thank you! <3

**Comment by ret2libc** [comment]
I have enabled auto-merge for this one. Let's see :)

**Pull request merged**